_posts/network/2023-12-20-create-ha-firewall.md

```diff
@@ -15,7 +15,7 @@ The tutorial does not include configuring the firewall.
 ## Requirements
 * A public IPv4 subnet where you can set allowed address pairs with at least three available IP addresses 
   * For example [Rent an IPv4 range]({{ '/articles/create-byoip-network' | relative_url }})
-* An internal network without a router and with at least three available IP addresses (Named `routed` in this tutorial)
+* An internal network without a router and with at least three available IP addresses (The router is named `OpenStack-internal-router` in this tutorial)
```

Comment:
Please change `OpenStack-internal-router` to `internal-network`

Reply:
Adjusted and comitted.